src/Centreon/ServiceProvider.php

```diff
@@ -94,15 +131,26 @@ public function register(Container $pimple): void
          * Services
          */
 
-        $pimple['centreon.broker_configuration_service'] = function(Container $container): BrokerConfigurationService {
+        $pimple['centreon.broker_configuration_service'] = function (Container $container): BrokerConfigurationService {
             $service = new BrokerConfigurationService($container['configuration_db']);
             $service->setBrokerInfoRepository($container['centreon.broker_info_repository']);
 
             return $service;
         };
+
+        $pimple['centreon.module_form_manager'] = function (Container $container) {
```

Comment:
please set 'centreon.module_form_manager' as a constant it will be more easy for managing and will reduce tech debt. You can see here a example https://github.com/centreon/centreon/blob/MON-3196-new-ext-manager/src/Centreon/ServiceProvider.php#L20